app/views/layouts/application.html.erb

```diff
@@ -3,8 +3,6 @@
 <head>
   <title><%= yield :title %> - GOV.UK</title>
   <!--[if gt IE 8]><!--><%= stylesheet_link_tag "application" %><!--<![endif]-->
-  <!--[if IE 6]><%= stylesheet_link_tag "application-ie6" %><script>var ieVersion = 6;</script><![endif]-->
-  <!--[if IE 7]><%= stylesheet_link_tag "application-ie7" %><script>var ieVersion = 7;</script><![endif]-->
```

Comment:
I know we don't support these browsers anymore but have you checked what effect removing these stylesheets has? I've faced a similar problem with the gov.uk homepage and have concluded that I'm not quite ready to break it for what turns out to be something like 40,000 users just yet.

Reply:
we can update `<!--[if gt IE 8]>` to `<!--[if lte IE 8]>`and serve those browsers that stylesheet and and any fixes in that stylesheet, plus import html5 shiv

we can't use the old 
```
$is-ie: true;
$ie-version: 8;
```
as govuk-frontend doesn't support it.

is there a better way while moving to govuk-frontend?

Reply:
Updating the conditional is a good idea.

Reply:
Wait, hang on, wouldn't updating the conditional like that be exactly the same as removing the conditional entirely? It's saying if the browser is IE8 or below then use this stylesheet, and every other browser will use it anyway. Might as well just remove it.

Reply:
not exactly. if you take away 6 and 7 stylesheets and don't change the ie8 conditiional, 6 & 7 get none of the application styles, just styles from govuk-template
<img width="1051" alt="Screen Shot 2019-09-05 at 12 06 29" src="https://user-images.githubusercontent.com/3758555/64336767-ada89100-cfd5-11e9-880a-09f1966a3aea.png">

Reply:
i've switched to `lte ie8` so 6 & 7 get the ie8 stylesheet

Reply:
@andysellick see comment with screenshot from IE6&7

Reply:
That all looks great. I'm still a bit confused about how it's working, but don't worry.